examples/StandAloneFileWatcher/README.md

```diff
@@ -0,0 +1,50 @@
+## Stand Alone File Watcher example agent
+
+This python script will listen to the specified files and publish updates to specific topics on the remote instance.
+
+Setup:
+1. Make sure volttron instance is running using tcp address. Use vcfg command to configure the volttron instance address.
+
+2. Create a public and secret key pair for this agent using this command:
+```
+vctl auth keypair
+```
+Update settings.py with your public and secret keys
+
+3. Create a public server key using this command: 
+```
+vctl auth serverkey
+```
+Update settings.py with this server key
+
+5. Update the config section of settings.py with the files you want this agent to watch for and
+which topics it should publish on for each file.  
+
+4. Add this standalone agent to volttron auth entry using this command:
+```
+vctl auth add
+```
+Provide the IP of the volttron instance when prompted for
+address[]: and  provide public key of standalone agent when prompted
```

Comment:
the ip address should be where the agent is running from not the instance's vip address.  they can leave it blank to accept all

Reply:
credentials should be the publickey generated above in the keypair command

Reply:
as a shortcut you could do vctl auth add --user_id identity_of_agent --credentials publickey_generated_above